README.md

```diff
@@ -11,9 +11,10 @@ Repository contains:
 # How to deploy
 ### Deploy master Buildbot
 Dependencies:
-- CentOS 7.3
-- python 3.6.x
+- CentOS v7.3
+- python v3.6.x
 - postgresql
+- git v1.7.10+ (for the `credentials.helper`)
```

Comment:
Maybe it's better to use latest version of git?

Reply:
Fixed